saltstack/base/salt/prerequisites/pip.sls

```diff
@@ -34,6 +34,12 @@ install_pyyaml:
     - name: pip install PyYAML --ignore-installed
     - unless: pip list | grep -E 'PyYAML'
 
+install_psycopg2:
```

Comment:
Why is this needed? We installing psycopg2 with yum now and with version 2.8.5

Reply:
This was added because of https://github.com/hortonworks/cloudbreak/blob/master/orchestrator-salt/src/main/resources/salt/salt/cloudera/agent/init.sls#L14

I don't see psycopg2 being installed anywhere in the image.

Reply:
cm agent install brings it as a package dependency

Reply:
Will remove it from here. Should I remove the section from agent/init.sls as well?

Reply:
no, that's for backward compatibility